crowbar_framework/app/models/provisioner_service.rb

```diff
@@ -179,76 +179,85 @@ def transition(inst, name, state)
   end
 
   def load_repository_bag
-    bag = Chef::DataBag.load("repositories") rescue nil
+    bag = Chef::DataBag.load("crowbar/repositories") rescue nil
```

Comment:
Style/RescueModifier: Avoid using `rescue` in its modifier form. (https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers)